packages/decoupler/meta.yaml

```diff
@@ -0,0 +1,13 @@
+name: decoupler
+description: |
+    Python package to infer biological activities from omics data using a collection of methods.
+project_home: https://github.com/saezlab/decoupler-py
+documentation_home: https://decoupler-py.readthedocs.io/en/latest/
+tutorials_home: https://decoupler-py.readthedocs.io/en/latest/
+install:
+    pypi: decoupler
+tags:
+    - TF
+    - GSEA
+    - Pathway
```

Comment:
Should the tags be acronyms?

Reply:
Until we define a controlled vocabulary, these are anyway just arbitrary examples showcasing that one can add tags to the yaml file :shrug:

Reply:
Yes, but should the examples that we start with be acronyms?